docs/index.rst

```diff
@@ -151,6 +151,20 @@ on a CI/build server).
 * ``TWINE_NON_INTERACTIVE`` - Do not interactively prompt for username/password
   if the required credentials are missing.
 
+Proxy Support
+^^^^^^^^^^^^^
+
+Twine allows you to upload your distributions with/without proxy via
+these environment variables:
+
+* ``HTTP_PROXY`` - Upload your distributions with HTTP proxy.
+* ``HTTPS_PROXY`` - Upload your distributions with HTTPS proxy.
+* ``NO_PROXY`` - Upload your distributions without proxy.
+
+For more information, please visit
+`"Proxies" section in requests docs <https://docs.python-requests.org/en/master/user/advanced/#proxies>`_
+or read `an article about proxy <https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/>`_.
+
```

Comment:
I think this should be rewritten to be provide a minimal example, and rely on the other documentation for details. Something like:

```rst
Twine can be configured to use a proxy by setting environment variables. For example:

.. code-block:: bash

   export HTTPS_PROXY=socks5://user:pass@host:port

   twine upload dist/*

For more information, see the Requests documentation on
`proxies <https://docs.python-requests.org/en/master/user/advanced/#proxies>`_ and
`SOCKS <https://docs.python-requests.org/en/master/user/advanced/#socks>`_ , and
`an in-depth article about proxy environment variables <https://about.gitlab.com/blog/2021/01/27/we-need-to-talk-no-proxy/>`_.
```

Reply:
Therefore, we don't have to document ``HTTP_PROXY``, ``HTTPS_PROXY`` and ``NO_PROXY`` in this section but giving an example instead? Or we should keep both of them there?